
---

bool success

---

string picked_object
